monitoring/testonly/delta.go

```diff
@@ -15,30 +15,45 @@
 package testonly
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/google/trillian/monitoring"
 )
 
-// CounterSnapshot records the values of a counter at a specific point in time.
+// CounterSnapshot records the value of a counter at a specific point in time.
```

Comment:
nit: Would `// ... records the value of one or more counters at ...` be more accurate? Or maybe `any number of counters`?

Reply:
I suppose it depends on whether you think of `Counter` as being one counter with many values, or as a collection of counters.

Reply:
From reading [the Prometheus docs](https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels), I think the correct interpretation is that it's one counter that contains one or more time series, each identified by a set of labels. I've simplified the code and reworded the documentation to reflect this.